public/js/games-client.js

```
@@ -432,249 +751,765 @@ class GameClient {
+    document.body.appendChild(notification);
+    
+    // Auto-remove after 10 seconds
+    setTimeout(() => {
+      if (notification.parentNode) {
+        notification.remove();
+      }
+    }, 10000);
+  }
 
-    buttonContainer.appendChild(leaveBtn);
+  // Update available games list
```

Biome (2.1.2)

[error] Duplicate class member name "showGameNotification"



(lint/suspicious/noDuplicateClassMembers)